Library/OcAudioLib/OcAudio.c

```diff
@@ -128,7 +128,7 @@ InternalMatchCodecDevicePath (
         &gEfiAudioIoProtocolGuid,
         (VOID **) &Private->AudioIo
         );
-      return EFI_SUCCESS;
+      return Status;
```

Comment:
This change looks good to me, but I guess it needs to be mentioned in the commit message at least. Marking not to forget.